x/pool-incentives/keeper/grpc_query.go

```diff
@@ -92,9 +93,13 @@ func (q Querier) IncentivizedPools(ctx context.Context, _ *types.QueryIncentiviz
 	}, nil
 }
 
-// ExternalIncentiveGauges iterates over all gauges, returns gauges externally incentivized,
-// excluding default gagues created with pool.
+// ExternalIncentiveGauges iterates over all gauges, returns gauges externally
+// incentivized, excluding default gauges created with pool.
 func (q Querier) ExternalIncentiveGauges(ctx context.Context, req *types.QueryExternalIncentiveGaugesRequest) (*types.QueryExternalIncentiveGaugesResponse, error) {
+	if req == nil {
+		return nil, status.Error(codes.InvalidArgument, "empty request")
```

Comment:
nit: I see this error message and a few others being reused several times, consider extracting constants